testing/travis-fork-check.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import logging
+import os
+import shlex
+import subprocess
+import sys
+
+def check_travis_repo_slug():
+    if 'encrypted_6a9d32f3e0bd_key' not in os.environ:
```

Comment:
That is one of the env vars that Travis added automatically for encrypting the CI private ssh key. I needed to make sure that the secrets were missing, so I picked out one of them. Is there a better way?
> On May 10, 2016, at 11:12, Steven Borrelli <notifications@github.com> wrote:
> 
> In testing/travis-fork-check.py <https://github.com/CiscoCloud/mantl/pull/1415#discussion_r62702330>:
> 
> > @@ -0,0 +1,27 @@
> > +#!/usr/bin/env python
> > +from __future__ import print_function
> > +import logging
> > +import os
> > +import shlex
> > +import subprocess
> > +import sys
> > +
> > +def check_travis_repo_slug():
> > +    if 'encrypted_6a9d32f3e0bd_key' not in os.environ:
> Why is this value hard-coded?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/CiscoCloud/mantl/pull/1415/files/b4bfa5710a9ad82d367b24ba56fe94fea2dbf924#r62702330>